web/controllers/screeningController/index.ts

```diff
@@ -405,4 +406,125 @@ export async function updateCourse(req: Request, res: Response) {
+                .createQueryBuilder(Student, "student")
+                .leftJoinAndSelect("student.screening", "screening")
+                .where("student.isInstructor = true AND screening.success = false AND (student.email ILIKE :search OR student.lastname ILIKE :search)", { search })
+                .take(20)
+                .getMany();
+        } else if (screeningStatus === ScreeningStatus.Unscreened) {
+            instructors = await getManager()
+                .createQueryBuilder(Student,"student")
+                .leftJoinAndSelect("student.screening", "screening")
+                .where("student.isInstructor = true AND screening.success is NULL AND (student.email ILIKE :search OR student.lastname ILIKE :search)", { search })
+                .take(20)
+                .getMany();
+        }
+
+
+        return res.json({ instructors });
+    } catch (error) {
+        logger.warn("/screening/instructors failed with", error.message);
+        return res.status(500).send("internal server error");
+    }
+}
+/**
+ * @api {POST} /screening/instructor/:instructorID/update updateInstructor
+ * @apiVersion 1.0.1
+ * @apiDescription
+ *
+ * Updates an instructor
+ *
+ *
+ * Only screeners with a valid token in the request header can use the API.
+ *
+ * @apiName updateCourse
+ * @apiGroup Screener
+ *
+ * @apiUse Authentication
+ *
+ * @apiExample {curl} Curl
+ * curl -k -i -X POST -H "Token: <AUTHTOKEN>" [host]/api/screening/instructor/id/update
+ *
+ * @apiParam (JSON Body) {boolean} isStudent the instructors can also be students at the same time
+ * @apiParam (JSON Body) {boolean} verified wether the instructor gets verified
+ * @apiParam (JSON Body) {string|undefined} phone sets the instructors phone number
+ * @apiParam (JSON Body) {Date|undefined} birthday sets the instructors birthday
+ * @apiParam (JSON Body) {string|undefined} commentScreener adds a comment to the screening
+ * @apiParam (JSON Body) {string|undefined} knowscsfrom
+ * @apiParam (JSON Body) {string|undefined} screenerEmail
+ * @apiParam (JSON Body) {string|undefined} subjects
+ * @apiParam (JSON Body) {string|undefined} feedback
+ */
+export async function updateInstructor(req: Request, res: Response) {
+    try {
+        const { id } = req.params;
+        const { isStudent } = req.body;
+        const screeningResult = new ApiScreeningResult(req.body);
+
+        if (typeof id !== "string" || !Number.isInteger(+id))
+            return res.status(400).send("Invalid instructor id!");
+
+        if (!screeningResult.isValid() || !(isStudent === true || isStudent === false))
+            return res.status(400).send("Invalid instructor update!");
+
+        const instructor = await getManager().findOne(Student, { where: { id: +id, isInstructor: true } });
+
+        if (!instructor)
+            return res.status(404).send("Instructor not found");
 
+        await instructor.addScreeningResult(screeningResult);
+
+        instructor.isStudent = isStudent;
```

Comment:
So if someone will be allowed to be instructor and student/tutor, he will have two screenings, one InstructorScreening and a normal Screening. This is how it is modeled in the backend right now.

Reply:
That's not what Christopher told us. Instructors can also be enabled as one on one with one checkbox.

Reply:
That's true what Christopher said from the user's perspective. For the screener it should just be an additional checkbox. But in the backend, the current modeling will not work this way. Therefore the screening somewhere needs to be duplicated.